### ISSUE TITLE
ICICI bank home page! Date is not able to select in the calendar.

### ISSUE THREAD

**Issue by nsingh2001** [nsingh2001/bicici#16]
UAT 2022-04
Windows 10
Chrome Version 100.0.4896.88 (Official Build) (64-bit)
NVDA 2021.3.1
Repro Steps
1. Open the URL  https://www.icicibank.com/
2.press H for heading till to reach to ‘I mobile -pay ’heading level 5and two times tab to reach tab ‘right product for you ‘
3. Reached to the tab ‘find the right product for you ‘
4.press tab  reached to combo box  and select ‘insurance’
5. press tab to reach next combo box and select ‘securing my loved one’
6.under the heading calculate your monthly premium press tab and then reached to the radio button 
7.After two radio buttons reached to the calendar
Expected Result - the calendar should be accessible to select the date
Actual Result- calendar is not able to select the date.
WCAG Guideline
1. https://www.w3.org/TR/WCAG21/#name-role-value

**Comment by manish10** [comment]
Hi Narendra, The expected and actual result are not clear in that I do not understand what is the problem in selecting the date. Can you be more descriptive about the problem?
Also, you need to use correct grammar in the title and body of the bug. Please try to rewrite this with correct grammar.
